src/Aspire.Hosting/ApplicationModel/DefaultLaunchProfileAnnotation.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+
+namespace Aspire.Hosting.ApplicationModel;
+
+[DebuggerDisplay("Type = {GetType().Name,nq}, LaunchProfileName = {LaunchProfileName}")]
+internal sealed class DefaultLaunchProfileAnnotation : IResourceAnnotation
```

Comment:
Does this need to be public? We've got behavior that sets this annotation when calling `CreateAsync(...)` but we don't have a mechanism to override it, and we can't modify the annotations because we don't have this type public.

Reply:
> Does this need to be public? 

What's the use case? I followed the pattern set by `LaunchProfileAnnotation` btw, which is internal.

> We've got behavior that sets this annotation when calling `CreateAsync(...)` but we don't have a mechanism to override it, and we can't modify the annotations because we don't have this type public.

Other than via configuration & env var?